gradle/libs.versions.toml

```
@@ -0,0 +1,8 @@
+[versions]
+dropwizard-metrics = "4.2.5"
+slf4j = "1.7.32"
```

I think we should use the versions from the platform and not the outdated one from the properties file.
```suggestion
dropwizard-metrics = "4.2.19"
slf4j = "2.0.7"
```